bom/pom.xml

```diff
@@ -41,7 +41,7 @@
     <netty.version>4.1.8.Final</netty.version>
     <objenesis.version>2.4</objenesis.version>
     <proton.version>0.18.0</proton.version>
-    <qpid-dispatch.version>0.7.0</qpid-dispatch.version>
+    <qpid-dispatch.version>0.8.0</qpid-dispatch.version>
```

Comment:
Yes. I was told it will be available any day now. That's why I used enmasse image for now for testing.